Src/AutoFixture.xUnit.net2.UnitTest/Scenario.cs

```diff
@@ -72,6 +72,42 @@ public void InlineAutoDataSuppliesDataSpecimensOnlyForNonProvidedValues(string s
             Assert.NotEqual("bar", s3);
         }
 
+        // This test and its associated types is used to document one of the
+        // InlineAutoDataAttribute constructor overloads.
+        [Theory]
+        [MyCustomInlineAutoData(1337)]
+        [MyCustomInlineAutoData(1337, 7)]
```

Comment:
The second inline data value (I.e 7, which corresponds to the `y` argument when the test method runs) has no assertion and is unused inside the method.

Instead of using the literal value `7`, we could use a `const` to give it a name.

I'm not sure what name would suit here.